### ISSUE TITLE
Serverless Stack tutorial showing usage of step functions instead of Lambda

### ISSUE THREAD

**Issue by GurpreetSingh5** [AnomalyInnovations/serverless-stack-com#400]
Additional chapters or totally new sub chapters can be started on this.
This would be widely helpful as AWS Step functions are equally used, and there is no proper stack oriented tutorial on that.

**Comment by jayair** [comment]
Yeah that's true. We'll look into adding a section when we get a chance.

**Comment by GurpreetSingh5** [comment]
Great. 
Have been already working on this.
Let me know, if open for contributions.

**Comment by jayair** [comment]
Yeah for sure. If you are interested in writing in a chapter, that would be great. Post a rough outline of what you want to cover and I can give you some feedback before started.
